modules/fb_apache/lib/logstash_registry.rb

```diff
@@ -0,0 +1 @@
+LogStash::PLUGIN_REGISTRY.add(:modules, "fb_apache", LogStash::Modules::Scaffold.new("fb_apache", File.join(File.dirname(__FILE__), "..", "configuration")))
```

Comment:
True.  It's also just for testing, so should we remove it?  It doesn't hurt anything being there...